frontend/public/components/alert-manager.tsx

```diff
@@ -38,7 +40,10 @@ const Details: React.SFC<DetailsProps> = (props) => {
             <dd>{spec.version}</dd>
             <dt>Replicas</dt>
             <dd>
-              <button type="button" className="btn btn-link co-modal-btn-link co-modal-btn-link--left" onClick={openReplicaCountModal}>{pluralize(spec.replicas, 'pod')}</button>
+              <button type="button" className="pf-c-button pf-m-link pf-m-inline" onClick={openReplicaCountModal}>
```

Comment:
Might be worth swapping to PF4 `<Button variant="link" isInline onClick={openReplicaCountModal}>` while touching all these instances.

Reply:
updated